pkgs/by-name/pr/proton-mail/package.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, dpkg
+, electron
+}
+:
+stdenv.mkDerivation rec {
+  pname = "proton-mail";
+  versionBase = "1.0.0";
+  versionSuffix = "beta";
+  version = "${versionBase}-${versionSuffix}";
+
+  src = fetchurl {
+    url = "https://github.com/ProtonMail/inbox-desktop/releases/download/v${versionBase}/${pname}_${versionBase}_amd64.deb";
```

```suggestion
    url = "https://github.com/ProtonMail/inbox-desktop/releases/download/v${versionBase}/protpn-mail_${versionBase}_amd64.deb";
```
See #277994. Also applies to other occurrences.

---

used helper (non pname) variable isntead to minimize the risk of this divering in different lines